cloudformation 2.0.7 release

pethers opened the pull request.

ganncamp commented on cloudformation.properties:
```diff
@@ -2,15 +2,21 @@ name=Cloudformation
 2.0.5.description=Support 6 new cfn-nag rules.
 2.0.5.date=2020-04-29
-2.0.5.sqVersions=[7.9,LATEST]
+2.0.5.sqVersions=[7.9,8.4]
```
You need to update this end range to 8.5.*. Otherwise users of the current SQ version won't be prompted to upgrade to your new version in-app.

ganncamp commented on cloudformation.properties:
```diff
@@ -2,15 +2,21 @@ name=Cloudformation
+2.0.7.description=Bugfixes, updated rule descriptions and nist 800-53 tags.
+2.0.7.date=2020-10-31
+2.0.7.sqVersions=[7.9,LATEST]
+2.0.7.changelogUrl=https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-2.0.7
```
Not required, but it would be nice in your changelog to link the 2.0.6 mention to its changelog - to give users an easy & transparent way to understand _all_ the changes in the version.

pethers commented:
Updated the version to 8.5.* and updated release notes in https://github.com/Hack23/sonar-cloudformation-plugin/releases/tag/sonar-cloudformation-plugin-2.0.7
